services/tenant-ui/frontend/src/matomoSetup.ts

```diff
@@ -0,0 +1,18 @@
+//@ts-ignore
+var _paq = (window._paq = window._paq || []);
+_paq.push(['trackPageView']);
+_paq.push(['enableLinkTracking']);
+function setup(MATOMO_URL: string) {
+	var u = `//${MATOMO_URL}/`;
+	_paq.push(['setTrackerUrl', u + 'matomo.php']);
+	_paq.push(['setSiteId', 1]);
+	var d = document,
+		g = d.createElement('script'),
+		s = d.getElementsByTagName('script')[0];
+	g.type = 'text/javascript';
+	g.async = true;
+	g.src = u + 'matomo.js';
+	//@ts-ignore
+	s.parentNode.insertBefore(g, s);
```

Comment:
does `parentNode?.` fix the ts complaint on this line?

Reply:
I just made `s` of type any. This way we will at least get an error if this some how went wrong. We handle the error in the main.ts